spec.html

```diff
@@ -7583,7 +7583,7 @@ <h1>SetFunctionLength ( _F_, _length_ )</h1>
       <emu-alg>
         1. Assert: _F_ is an extensible object that does not have a `"length"` own property.
         1. Assert: Type(_length_) is Number.
-        1. Assert: _length_ ≥ 0 and ! IsInteger(_length_).
+        1. Assert: _length_ ≥ 0 and ! IsInteger(_length_) is *true*.
```

Comment:
Let's also remove non-ASCII `≥` (the only occurrence in spec).

```suggestion
        1. Assert: _length_ &ge; 0 and ! IsInteger(_length_) is *true*.
```

Reply:
Fine with me, but I'll wait for an editor's nod.

Reply:
Since ecmarkup doesn't (unfortunately) automatically convert `≥` to `&gte;`, the HTML version is probably the better approach for now.

Reply:
(I'm not sure why that makes a difference, but I'll take it.)

Reply:
more that i'd expect the HTML encoding, for wider compatibility - but in terms of consistency, it seems like we use the encoded form way more than the nice, pretty, character form.